### PR TITLE
Allow objects in locations for parameterless paths

### PR DESCRIPTION
**Pull request by t-kameyama**
Fix for #165

### CODE REVIEW

File: ktor-features/ktor-locations/test/org/jetbrains/ktor/tests/locations/LocationsTest.kt [hunk 1]
```diff
@@ -341,5 +341,74 @@ class LocationsTest {
...
+            get<help> {
+                call.respond(HttpStatusCode.OK)
+            }
+        }
+        urlShouldBeHandled(href)
+        urlShouldBeUnhandled("/help/123")
+    }
+
+    @location("/users")
+    object users {
```
Comment (kosiakk) [inline ktor-features/ktor-locations/test/org/jetbrains/ktor/tests/locations/LocationsTest.kt]: A very realistic example! 👍

**Comment by spand** [comment]
Does it throw an error on ie `@location("/users/{id}") ? Should it ?

**Comment by orangy** [comment]
It should. There is no place to bind `{id}`

**Comment by orangy** [comment]
Thank you!

**Pull request merged**